server/task_executor_pitr_cutover.go

```diff
@@ -94,24 +100,17 @@ func (exec *PITRCutoverTaskExecutor) pitrCutover(ctx context.Context, task *api.
 	if err != nil {
```

Comment:
Use `defer conn.Close()` here? 
> [Every Conn must be returned to the database pool after use by calling Conn.Close.](https://pkg.go.dev/database/sql#DB.Conn)

Reply:
Thought of that but we cannot do that.
The `conn` will be used in an asynchronous goroutine, which means the connection should be used after this function exits.

Reply:
Tried to fix. PTAL.